src/router/index.ts

```diff
@@ -9,6 +11,43 @@ const routes: Array<RouteConfig> = [
         path: '/',
         name: 'Home',
         component: Home,
+        meta: {
+            linkText: 'Home',
+            title: 'Guide Vue - Home',
+            metaTags: [
+                {
+                    name: 'description',
+                    content:
+                        'The home page of Open Slot guide vue web application.',
+                },
+                {
+                    property: 'og:description',
+                    content:
+                        'The home page of Open Slot guide vue web application.',
+                },
+            ],
+        },
+    },
+    {
+        path: '/about',
+        name: 'About',
+        component: About,
+        meta: {
+            linkText: 'About',
+            title: 'Guide Vue - About',
+            metaTags: [
+                {
+                    name: 'description',
+                    content:
+                        'The about page of Open Slot guide vue web application.',
+                },
+                {
+                    property: 'og:description',
+                    content:
+                        'The about page of Open Slot guide vue web application.',
+                },
+            ],
+        },
```

Comment:
@Andresws12 Here I can not do a review because I haven´t learn about this. In the next daily I will ask about this.